library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -664,17 +669,15 @@ libspdm_return_t libspdm_build_response(void *spdm_context, const uint32_t *sess
                                                               &get_info->chunk_handle,
                                                               &my_response_size, my_response);
         }
-        else
-        #endif /* LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP */
-        {
-            LIBSPDM_DEBUG((LIBSPDM_DEBUG_ERROR,
-                           "Warning: Could not save chunk. Scratch buffer too small.\n"));
+        #else
+        LIBSPDM_DEBUG((LIBSPDM_DEBUG_ERROR,
+                       "Warning: Could not save chunk. Scratch buffer too small.\n"));
 
-            status = libspdm_generate_error_response(context,
-                                                     SPDM_ERROR_CODE_LARGE_RESPONSE,
-                                                     spdm_request->request_response_code,
-                                                     &my_response_size, my_response);
-        }
+        status = libspdm_generate_error_response(context,
+                                                 SPDM_ERROR_CODE_LARGE_RESPONSE,
+                                                 spdm_request->request_response_code,
```

Comment:
`ResponseLarge` ERROR response message has its `ExtendedErrorData`, so I think here `libspdm_generate_error_response()` function should be changed to `libspdm_generate_extended_error_response()` function.

Reply:
Thanks. I have changed `libspdm_generate_error_response()` function should be changed to `libspdm_generate_extended_error_response()`.